src/components/button/cva.ts

```diff
@@ -0,0 +1,40 @@
+import { type VariantProps, cva } from 'class-variance-authority';
+
+export type BaseButtonProps = VariantProps<typeof button>;
```

Comment:
move this below the button variable declaration

Reply:
done